src/Adaptivity/AdaptedTriangulations.jl

```diff
@@ -0,0 +1,316 @@
+
+"""
+  AdaptedTriangulation
+
+  Triangulation produced from an AdaptedDiscreteModel.
+  
+  Contains: 
+
+  - adapted_model ~> AdaptedDiscreteModel for the triangulation.
+  - trian ~> Triangulation for the background model, i.e get_model(adapted_model).
+"""
+struct AdaptedTriangulation{Dc,Dp,A<:Triangulation{Dc,Dp},B<:AdaptedDiscreteModel} <: Triangulation{Dc,Dp}
+  trian::A
+  adapted_model::B
+
+  function AdaptedTriangulation(trian::Triangulation{Dc,Dp},model::AdaptedDiscreteModel{Dc2,Dp}) where {Dc,Dc2,Dp}
+    @check !isa(trian,AdaptedTriangulation)
+    @check Dc <= Dc2
+    A = typeof(trian)
+    B = typeof(model)
+    return new{Dc,Dp,A,B}(trian,model)
+  end
+end
+
+function get_adapted_model(t::AdaptedTriangulation)
+  return t.adapted_model
+end
+
+function get_adaptivity_glue(t::AdaptedTriangulation)
+  return get_adaptivity_glue(get_adapted_model(t))
+end
+
+# Relationships
+function is_child(t1::AdaptedTriangulation,t2::Triangulation)
+  return is_child(get_adapted_model(t1),get_background_model(t2))
+end
+
+function is_child(t1::AdaptedTriangulation,t2::AdaptedTriangulation)
+  return is_child(get_adapted_model(t1),get_adapted_model(t2))
+end
+
+is_child(t1::Triangulation,t2::AdaptedTriangulation) = false
+
+is_related(t1::AdaptedTriangulation,t2::Triangulation) = is_child(t1,t2)
```

Comment:
Same comment as above (`get_parent`, etc.) but with Triangulations instead of DiscreteModels ...

Reply:
Addressed in [bf33178](https://github.com/gridap/Gridap.jl/pull/838/commits/bf33178f74e081bd82c0d8ce0a15edf2b212d83b)